src/findReadMe.ts

```diff
@@ -19,6 +19,14 @@ export const findReadMe = async (dir: string): Promise<string | undefined> => {
         if (newDir === dir) {
             return undefined
         }
+        const url = vfs.urlParse(newDir)
+        if (url !== undefined) {
+            const pathSplit = url.path.split("/")
+            if (pathSplit.length === 2 && pathSplit[0] === "github.com") {
+                // return undefined if it's a GitHub organization instead of a repository
+                return undefined
```

Comment:
can we log here or is logging not available?

Reply:
It's not available at this point. We need to think about solution here.

Reply:
As discussed, we may have another library `@ts-common/log` which will define common interface and provide logger instance. The library should wrap one of the existing library.